recipes/prismatic/meta.yaml

```diff
@@ -0,0 +1,164 @@
+
+build:
+  skip: True  # [win or py2k]
+  number: {{ build }}
+  string: "{{processor}}_h{{ PKG_HASH }}_{{ build }}"
+
+outputs:
+  - name: prismatic
+    build:
+      number: {{ build }}
+      skip: True  # [win or py2k]
+      string: "{{processor}}_h{{ PKG_HASH }}_{{ build }}"
+    requirements:
+      run:
+        - {{ pin_subpackage('prismatic_cli', exact=True) }}
+        - {{ pin_subpackage('prismatic_gui', exact=True) }}
+        - {{ pin_subpackage('pyprismatic', exact=True) }}  # [not osx]
+
+  - name: prismatic_cli
+    build:
+      skip: True  # [win]
+      string: "{{processor}}_h{{ PKG_HASH }}_{{ build }}"
+      script: bash ${RECIPE_DIR}/combined_build_install.sh cli # [not win]
+
+    requirements:
+      build:
+        - {{ compiler('c') }}                
+        - {{ compiler('cxx') }}              
+        - {{ cdt('libxext-devel') }}         # [linux]
+        - {{ cdt('mesa-libgl-devel') }}      # [linux]
+        - {{ cdt('mesa-dri-drivers') }}      # [linux]
+        - {{ cdt('libxau-devel') }}          # [linux]
+        - {{ cdt('libselinux-devel') }}      # [linux]
+        - {{ cdt('libxdamage') }}            # [linux]
+        - {{ cdt('libxfixes') }}             # [linux]
+        - {{ cdt('libxxf86vm') }}            # [linux]
+        - make                               # [unix]
+        - cmake
+      host:
+        - boost
+        - hdf5
+        - fftw
+        - zlib
+      run:
+        - zlib
+        - hdf5
+        - fftw
+
+
+    test:
+      source_files:
+        - SI100.XYZ
+      commands:
+        - prismatic -i SI100.XYZ
+
+
+  - name: prismatic_gui
+    build:
+      skip: True  # [win]
+      string: "{{processor}}_h{{ PKG_HASH }}_{{ build }}"
+      script: bash ${RECIPE_DIR}/combined_build_install.sh gui # [not win]
+
+    requirements:
+      build:
+        - {{ compiler('c') }}                
+        - {{ compiler('cxx') }}              
+        - {{ cdt('libxext-devel') }}         # [linux]
+        - {{ cdt('mesa-libgl-devel') }}      # [linux]
+        - {{ cdt('mesa-dri-drivers') }}      # [linux]
+        - {{ cdt('libxau-devel') }}          # [linux]
+        - {{ cdt('libselinux-devel') }}      # [linux]
+        - {{ cdt('libxdamage') }}            # [linux]
+        - {{ cdt('libxfixes') }}             # [linux]
+        - {{ cdt('libxxf86vm') }}            # [linux]
+        - make                               # [unix]
+        - cmake
+      host:
+        - boost
+        - hdf5
+        - fftw
+        - qt 5.9.*
+        - h5py
+        - zlib
+      run:
+        - zlib
+      
+    test:
+      commands:
+        - command -v prismatic-gui  # [linux]
+        - test -f $PREFIX/Applications/prismatic-gui.app/Contents/MacOS/prismatic-gui  # [osx]
+
+  - name: pyprismatic
+    build:
+      skip: True  # [win or py2k or osx]
+      string: "{{processor}}_h{{ PKG_HASH }}_{{ build }}"
+      script: bash ${RECIPE_DIR}/combined_build_install.sh py # [not win]
```

Comment:
```suggestion
      script: bash ${RECIPE_DIR}/combined_build_install.sh py  # [not win]
```

Reply:
ahh, that solves the linter problem - thanks!